2014-09-24 15:22:24 Error: [MissingActionException] Action BlocksController::view() could not be found.
Exception Attributes: array (
  'controller' => 'BlocksController',
  'action' => 'view',
)
Request URL: /checklist/blocks/view/2
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(BlocksController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-25 09:07:55 Error: [MissingActionException] Action ChecklistsController::view() could not be found.
Exception Attributes: array (
  'controller' => 'ChecklistsController',
  'action' => 'view',
)
Request URL: /checklist/checklists/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-25 09:30:41 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 79]
2014-09-25 09:30:41 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 09:31:08 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 79]
2014-09-25 09:31:08 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 09:32:18 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 79]
2014-09-25 09:32:18 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 79, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 09:33:26 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Checklist'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT DISTINCT...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(79): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 09:43:45 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Checklist'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(79): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 09:44:38 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(79): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 09:45:29 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(79): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 10:28:53 Error: Fatal Error (4): syntax error, unexpected '$this' (T_VARIABLE) in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 62]
2014-09-25 10:28:53 Error: [FatalErrorException] syntax error, unexpected '$this' (T_VARIABLE)
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 11:10:01 Error: Fatal Error (4): syntax error, unexpected ';', expecting ')' in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 68]
2014-09-25 11:10:01 Error: [FatalErrorException] syntax error, unexpected ';', expecting ')'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 68)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 68, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 68, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 11:10:29 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'users INNER User.id = Checklist.user_id  WHERE `Classroom`.`id` = 1    LIMIT 1' at line 1
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(68): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:10:30 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'users INNER User.id = Checklist.user_id  WHERE `Classroom`.`id` = 1    LIMIT 1' at line 1
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(68): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:13:42 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'users INNER (User.id = Checklist.user_id)  WHERE `Classroom`.`id` = 1    LIMIT 1' at line 1
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(68): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:13:42 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'users INNER (User.id = Checklist.user_id)  WHERE `Classroom`.`id` = 1    LIMIT 1' at line 1
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(68): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:16:27 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Checklist'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(69): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:19:07 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Checklist'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(70): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:22:20 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(70): Model->find('first', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:26:27 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(70): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:28:40 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(70): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:29:38 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(70): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:31:07 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(71): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:37:50 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Checklist.user_id' in 'on clause'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT * FROM `...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT * FROM `...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT * FROM `...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(71): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 11:38:26 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'checklists.user_id' in 'on clause'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT * FROM `...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT * FROM `...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT * FROM `...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(71): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:09:45 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'checklists.user_id' in 'on clause'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT * FROM `...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT * FROM `...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT * FROM `...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(76): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:11:21 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'checklists.user_id' in 'on clause'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT * FROM `...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT * FROM `...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT * FROM `...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(76): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:30:21 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(71): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:39:09 Error: Fatal Error (1): Call to a member function find() on a non-object in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 71]
2014-09-25 14:39:09 Error: [FatalErrorException] Call to a member function find() on a non-object
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 71)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 71, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 71, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 14:49:21 Error: Fatal Error (4): syntax error, unexpected ',' in [C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php, line 74]
2014-09-25 14:49:21 Error: [FatalErrorException] syntax error, unexpected ','
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 14:49:47 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(77): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:50:27 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(78): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:50:57 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'checklists'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Classroom), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ClassroomsController.php(78): Model->find('all', Array)
#7 [internal function]: ClassroomsController->admin_view('1')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ClassroomsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ClassroomsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-25 14:57:57 Error: Fatal Error (4): syntax error, unexpected '>' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 3]
2014-09-25 14:57:57 Error: [FatalErrorException] syntax error, unexpected '>'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:14:26 Error: Fatal Error (4): syntax error, unexpected 'endif' (T_ENDIF) in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 79]
2014-09-25 15:14:26 Error: [FatalErrorException] syntax error, unexpected 'endif' (T_ENDIF)
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 79)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 79, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 79, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:20:00 Error: Fatal Error (4): syntax error, unexpected 'endforeach' (T_ENDFOREACH) in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 74]
2014-09-25 15:20:00 Error: [FatalErrorException] syntax error, unexpected 'endforeach' (T_ENDFOREACH)
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 74, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:20:50 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 69]
2014-09-25 15:20:50 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 69)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 69, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 69, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:23:47 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 70]
2014-09-25 15:23:47 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 70)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 70, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 70, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:26:11 Error: Fatal Error (4): syntax error, unexpected '<' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 3]
2014-09-25 15:26:11 Error: [FatalErrorException] syntax error, unexpected '<'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:32:22 Error: Fatal Error (4): syntax error, unexpected '<' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 3]
2014-09-25 15:32:22 Error: [FatalErrorException] syntax error, unexpected '<'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 3, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:44:42 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 65]
2014-09-25 15:44:42 Error: [FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 65)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 65, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 65, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-25 15:45:17 Error: Fatal Error (4): syntax error, unexpected ''admin'' (T_CONSTANT_ENCAPSED_STRING), expecting ')' in [C:\xampp\htdocs\checklist\app\View\Classrooms\admin_view.ctp, line 62]
2014-09-25 15:45:17 Error: [FatalErrorException] syntax error, unexpected ''admin'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'
Request URL: /checklist/admin/classrooms/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 62, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 08:41:04 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Classroom.id' in 'field list'
Request URL: /checklist/admin/users/view/4
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\UsersController.php(197): Model->find('all', Array)
#7 [internal function]: UsersController->admin_view('4')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-26 08:41:30 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Classroom'
Request URL: /checklist/admin/users/view/4
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\UsersController.php(197): Model->find('all', Array)
#7 [internal function]: UsersController->admin_view('4')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-26 08:45:45 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Classroom.id' in 'on clause'
Request URL: /checklist/admin/users/view/4
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\UsersController.php(203): Model->find('all', Array)
#7 [internal function]: UsersController->admin_view('4')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-26 09:37:47 Error: [NotFoundException] Invalid user
Request URL: /checklist/users/view/2
Stack Trace:
#0 [internal function]: UsersController->view('2')
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:38:29 Error: [MissingActionException] Action UsersController::edit() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'edit',
)
Request URL: /checklist/users/edit/3
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-26 09:38:57 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/view/2
Stack Trace:
#0 [internal function]: UsersController->admin_view('2')
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:39:33 Error: [MissingActionException] Action UsersController::admin_teste() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'admin_teste',
)
Request URL: /checklist/admin/users/teste
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-26 09:40:52 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:42:04 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:42:05 Error: Fatal Error (4): syntax error, unexpected '$mailto' (T_VARIABLE) in [C:\xampp\htdocs\checklist\app\View\Errors\error400.ctp, line 34]
2014-09-26 09:42:05 Error: [FatalErrorException] syntax error, unexpected '$mailto' (T_VARIABLE)
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 09:42:32 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:42:32 Error: Fatal Error (4): syntax error, unexpected '$mailto' (T_VARIABLE) in [C:\xampp\htdocs\checklist\app\View\Errors\error400.ctp, line 34]
2014-09-26 09:42:32 Error: [FatalErrorException] syntax error, unexpected '$mailto' (T_VARIABLE)
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 34, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 09:43:10 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:43:10 Error: Fatal Error (4): syntax error, unexpected '$mailto' (T_VARIABLE) in [C:\xampp\htdocs\checklist\app\View\Errors\error400.ctp, line 35]
2014-09-26 09:43:10 Error: [FatalErrorException] syntax error, unexpected '$mailto' (T_VARIABLE)
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 09:44:48 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:45:51 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:49:03 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:53:43 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:54:11 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:54:19 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 09:54:47 Error: [NotFoundException] Invalid user
Request URL: /checklist/admin/users/edit/
Stack Trace:
#0 [internal function]: UsersController->admin_edit()
#1 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2014-09-26 10:11:48 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:11:48 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:11:52 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:11:52 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:12:06 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:12:06 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:12:28 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:12:28 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:12:31 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:12:31 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:12:32 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:12:32 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:15:07 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:15:07 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:15:37 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:15:37 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/users/login
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:15:43 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:15:43 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:15:46 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:15:46 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:15:50 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:15:50 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:16:01 Error: Fatal Error (4): syntax error, unexpected ')' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 31]
2014-09-26 10:16:01 Error: [FatalErrorException] syntax error, unexpected ')'
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 31, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:16:43 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 35]
2014-09-26 10:16:43 Error: [FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:16:49 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 35]
2014-09-26 10:16:49 Error: [FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /checklist/admin
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:16:51 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 35]
2014-09-26 10:16:51 Error: [FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:17:03 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 35]
2014-09-26 10:17:03 Error: [FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /checklist/users/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 35, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 10:24:33 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 356]
2014-09-26 10:24:33 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /checklist/users/login
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 356)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 356, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 356, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 15:08:45 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ch...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `ch...')
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(1854): DboSource->create(Object(User), Array, Array)
#4 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(142): Model->save(Array)
#5 [internal function]: ChecklistsController->report()
#6 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#7 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#9 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-26 15:51:15 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('INSERT INTO `ch...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1012): DboSource->execute('INSERT INTO `ch...')
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(1854): DboSource->create(Object(User), Array, Array)
#4 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(142): Model->save(Array)
#5 [internal function]: ChecklistsController->report()
#6 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#7 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#9 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-26 16:47:05 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 130]
2014-09-26 16:47:05 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /checklist/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 16:47:34 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 130]
2014-09-26 16:47:34 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /checklist/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 130, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-26 16:57:08 Error: Fatal Error (4): syntax error, unexpected 'else' (T_ELSE) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 170]
2014-09-26 16:57:08 Error: [FatalErrorException] syntax error, unexpected 'else' (T_ELSE)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 08:02:42 Error: Fatal Error (4): syntax error, unexpected 'else' (T_ELSE) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 170]
2014-09-27 08:02:42 Error: [FatalErrorException] syntax error, unexpected 'else' (T_ELSE)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 08:03:44 Error: Fatal Error (4): syntax error, unexpected 'else' (T_ELSE) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 170]
2014-09-27 08:03:44 Error: [FatalErrorException] syntax error, unexpected 'else' (T_ELSE)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 170, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 08:04:14 Error: Fatal Error (1): Call to undefined method ChecklistsController::setflash() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 172]
2014-09-27 08:04:14 Error: [FatalErrorException] Call to undefined method ChecklistsController::setflash()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 172)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 172, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 172, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 08:48:21 Error: Fatal Error (1): Call to undefined method SessionComponent::set() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 147]
2014-09-27 08:48:21 Error: [FatalErrorException] Call to undefined method SessionComponent::set()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 147)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 147, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 147, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 09:26:01 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\xampp\htdocs\checklist\app\View\Checklists\report.ctp, line 80]
2014-09-27 09:26:01 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 80)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 80, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 80, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 09:26:41 Error: Fatal Error (4): syntax error, unexpected '<' in [C:\xampp\htdocs\checklist\app\View\Checklists\report.ctp, line 7]
2014-09-27 09:26:41 Error: [FatalErrorException] syntax error, unexpected '<'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 7)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 7, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 7, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 10:07:05 Error: Fatal Error (1): Call to undefined method ChecklistsController::_refreshAuth() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 158]
2014-09-27 10:07:05 Error: [FatalErrorException] Call to undefined method ChecklistsController::_refreshAuth()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 158)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 158, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 158, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 10:11:25 Error: [MissingActionException] Action ChecklistsController::logout() could not be found.
Exception Attributes: array (
  'controller' => 'ChecklistsController',
  'action' => 'logout',
)
Request URL: /checklist/checklists/logout
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-27 10:27:13 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `User`.`...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `User`.`...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('list', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(172): Model->find('list', Array)
#7 [internal function]: ChecklistsController->report()
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-27 10:29:07 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '_refreshAuth' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('_refreshAuth', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('_refreshAuth', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(609): DboSource->fetchAll('_refreshAuth', Array, Array)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(808): DboSource->query('_refreshAuth', Array, Object(User))
#5 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(158): Model->__call('_refreshAuth', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(158): User->_refreshAuth()
#7 [internal function]: ChecklistsController->report()
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-27 10:33:33 Error: Fatal Error (4): syntax error, unexpected 'public' (T_PUBLIC) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 346]
2014-09-27 10:33:33 Error: [FatalErrorException] syntax error, unexpected 'public' (T_PUBLIC)
Request URL: /checklist/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 346)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 346, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 346, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 10:46:09 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `User`.`...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `User`.`...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('list', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(172): Model->find('list', Array)
#7 [internal function]: ChecklistsController->report()
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-27 11:03:28 Error: Fatal Error (4): syntax error, unexpected ';' in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 149]
2014-09-27 11:03:28 Error: [FatalErrorException] syntax error, unexpected ';'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 149)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 149, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 149, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:04:38 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '11:09:38  WHERE `id` = 'id = 1'' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('UPDATE `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\Database\Mysql.php(414): DboSource->execute('UPDATE `checkli...')
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2585): Mysql->update(Object(User), Array, NULL, Array)
#4 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(149): Model->updateAll(Array, Array)
#5 [internal function]: ChecklistsController->report()
#6 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#7 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#9 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-27 11:05:37 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '11:09:37  WHERE `id` = 1' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('UPDATE `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\Database\Mysql.php(414): DboSource->execute('UPDATE `checkli...')
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2585): Mysql->update(Object(User), Array, NULL, Array)
#4 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(149): Model->updateAll(Array, Array)
#5 [internal function]: ChecklistsController->report()
#6 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#7 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#9 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-27 11:06:38 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '11:09:38  WHERE `User`.`id` = 1' at line 1
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('UPDATE `checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\Database\Mysql.php(414): DboSource->execute('UPDATE `checkli...')
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2585): Mysql->update(Object(User), Array, NULL, Array)
#4 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(149): Model->updateAll(Array, Array)
#5 [internal function]: ChecklistsController->report()
#6 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#7 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#8 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#9 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2014-09-27 11:47:29 Error: Fatal Error (1): Call to undefined method ChecklistsController::_refreshAuth() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 156]
2014-09-27 11:47:29 Error: [FatalErrorException] Call to undefined method ChecklistsController::_refreshAuth()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:48:04 Error: Fatal Error (1): Call to undefined function _refreshAuth() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 156]
2014-09-27 11:48:04 Error: [FatalErrorException] Call to undefined function _refreshAuth()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:49:09 Error: Fatal Error (4): syntax error, unexpected 'function' (T_FUNCTION) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 354]
2014-09-27 11:49:09 Error: [FatalErrorException] syntax error, unexpected 'function' (T_FUNCTION)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:49:31 Error: Fatal Error (4): syntax error, unexpected 'private' (T_PRIVATE) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 354]
2014-09-27 11:49:31 Error: [FatalErrorException] syntax error, unexpected 'private' (T_PRIVATE)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:50:06 Error: Fatal Error (4): syntax error, unexpected 'public' (T_PUBLIC) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 354]
2014-09-27 11:50:06 Error: [FatalErrorException] syntax error, unexpected 'public' (T_PUBLIC)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 354, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:50:50 Error: Fatal Error (4): syntax error, unexpected 'protect' (T_STRING), expecting function (T_FUNCTION) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 355]
2014-09-27 11:50:51 Error: [FatalErrorException] syntax error, unexpected 'protect' (T_STRING), expecting function (T_FUNCTION)
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 355)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 355, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 355, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:51:22 Error: Fatal Error (1): Call to undefined function _refreshAuth() in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 156]
2014-09-27 11:51:22 Error: [FatalErrorException] Call to undefined function _refreshAuth()
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 156, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 11:57:54 Error: Fatal Error (1): Call to private method AppController::_refreshAuth() from context 'ChecklistsController' in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 156]
2014-09-27 11:57:54 Error: [FatalErrorException] Call to private method AppController::_refreshAuth() from context 'ChecklistsController'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Call to private...', 'C:\\xampp\\htdocs...', 156)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to private...', 'C:\\xampp\\htdocs...', 156, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to private...', 'C:\\xampp\\htdocs...', 156, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:00:02 Error: Fatal Error (4): syntax error, unexpected '.', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 118]
2014-09-27 12:00:02 Error: [FatalErrorException] syntax error, unexpected '.', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:03:07 Error: Fatal Error (4): syntax error, unexpected '[', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 119]
2014-09-27 12:03:07 Error: [FatalErrorException] syntax error, unexpected '[', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 119)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 119, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 119, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:03:27 Error: Fatal Error (1): Cannot use object of type AuthComponent as array in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 118]
2014-09-27 12:03:27 Error: [FatalErrorException] Cannot use object of type AuthComponent as array
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118)
#1 [internal function]: ErrorHandler::handleError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:04:12 Error: Fatal Error (1): Cannot use object of type AuthComponent as array in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 118]
2014-09-27 12:04:12 Error: [FatalErrorException] Cannot use object of type AuthComponent as array
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118)
#1 [internal function]: ErrorHandler::handleError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 118, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:04:45 Error: Fatal Error (1): Cannot use object of type AuthComponent as array in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 119]
2014-09-27 12:04:45 Error: [FatalErrorException] Cannot use object of type AuthComponent as array
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 119)
#1 [internal function]: ErrorHandler::handleError(1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 119, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Cannot use obje...', 'C:\\xampp\\htdocs...', 119, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-27 12:08:03 Error: Fatal Error (4): syntax error, unexpected '[', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in [C:\xampp\htdocs\checklist\app\Controller\AppController.php, line 118]
2014-09-27 12:08:03 Error: [FatalErrorException] syntax error, unexpected '[', expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'
Request URL: /checklist/checklists/report
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 118, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-29 13:11:19 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php, line 458]
2014-09-29 13:11:19 Error: [FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /checklist/admin/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 458)
#1 [internal function]: ErrorHandler::handleError(1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 458, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 458, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-29 13:11:27 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\xampp\htdocs\checklist\app\Model\User.php, line 129]
2014-09-29 13:11:27 Error: [FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /checklist/admin/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 129)
#1 [internal function]: ErrorHandler::handleError(1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 129, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Maximum executi...', 'C:\\xampp\\htdocs...', 129, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 10:25:48 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1066 Not unique table/alias: 'Block'
Request URL: /checklist/admin/users/view/5
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\UsersController.php(208): Model->find('all', Array)
#7 [internal function]: UsersController->admin_view('5')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 10:27:41 Error: [PDOException] SQLSTATE[42S02]: Base table or view not found: 1051 Unknown table 'Block'
Request URL: /checklist/admin/users/view/5
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Checkli...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Checkli...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Checkli...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(User), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\UsersController.php(209): Model->find('all', Array)
#7 [internal function]: UsersController->admin_view('5')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 11:04:51 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\xampp\htdocs\checklist\app\Controller\UsersController.php, line 214]
2014-09-30 11:04:51 Error: [FatalErrorException] syntax error, unexpected '}'
Request URL: /checklist/admin/users/view/5
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 214)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 214, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 214, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 11:40:15 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'checklist.blocks.id' in 'where clause'
Request URL: /checklist/admin/checklists/view/13
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Block`....', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Block`....', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Block`....', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Block), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(214): Model->find('first', Array)
#7 [internal function]: ChecklistsController->admin_view('13')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 11:40:54 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Blocks.id' in 'where clause'
Request URL: /checklist/admin/checklists/view/13
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Block`....', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Block`....', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Block`....', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Block), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(214): Model->find('first', Array)
#7 [internal function]: ChecklistsController->admin_view('13')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 11:40:58 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Blocks.id' in 'where clause'
Request URL: /checklist/admin/checklists/view/13
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT `Block`....', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Block`....', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT `Block`....', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Block), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('first', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(214): Model->find('first', Array)
#7 [internal function]: ChecklistsController->admin_view('13')
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 11:42:36 Error: Fatal Error (4): syntax error, unexpected 'echo' (T_ECHO) in [C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php, line 216]
2014-09-30 11:42:36 Error: [FatalErrorException] syntax error, unexpected 'echo' (T_ECHO)
Request URL: /checklist/admin/checklists/view/13
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 216)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 216, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 216, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 11:59:14 Error: [MissingActionException] Action BlocksController::view() could not be found.
Exception Attributes: array (
  'controller' => 'BlocksController',
  'action' => 'view',
)
Request URL: /checklist/blocks/view/2
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(BlocksController), Object(CakeRequest))
#2 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2014-09-30 12:06:18 Error: Fatal Error (4): syntax error, unexpected ')', expecting ',' or ';' in [C:\xampp\htdocs\checklist\app\View\Users\view.ctp, line 88]
2014-09-30 12:06:18 Error: [FatalErrorException] syntax error, unexpected ')', expecting ',' or ';'
Request URL: /checklist/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 88)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 88, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 88, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 12:56:20 Error: Fatal Error (1): Unsupported operand types in [C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php, line 848]
2014-09-30 12:56:20 Error: [FatalErrorException] Unsupported operand types
Request URL: /checklist/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848)
#1 [internal function]: ErrorHandler::handleError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 12:56:21 Error: Fatal Error (1): Unsupported operand types in [C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php, line 848]
2014-09-30 12:56:21 Error: [FatalErrorException] Unsupported operand types
Request URL: /checklist/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848)
#1 [internal function]: ErrorHandler::handleError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 12:56:42 Error: Fatal Error (1): Unsupported operand types in [C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php, line 848]
2014-09-30 12:56:42 Error: [FatalErrorException] Unsupported operand types
Request URL: /checklist/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848)
#1 [internal function]: ErrorHandler::handleError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 12:56:45 Error: Fatal Error (1): Unsupported operand types in [C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php, line 848]
2014-09-30 12:56:45 Error: [FatalErrorException] Unsupported operand types
Request URL: /checklist/users/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848)
#1 [internal function]: ErrorHandler::handleError(1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Unsupported ope...', 'C:\\xampp\\htdocs...', 848, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2014-09-30 13:28:00 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'AND `User`.`shift` <> 'Tarde' AND `Checklist`.`created` >= '2014-09-29 00:00:00'' at line 1
Request URL: /checklist/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT DISTINCT...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Checklist), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(66): Model->find('all', Array)
#7 [internal function]: ChecklistsController->index()
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 13:30:12 Error: [PDOException] SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'AND `User`.`shift` <> 'Tarde' AND `Checklist`.`created` >= '2014-09-29 00:00:00'' at line 1
Request URL: /checklist/
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(458): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(424): DboSource->_execute('SELECT DISTINCT...', Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT DISTINCT...', Array, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Model\Datasource\DboSource.php(1106): DboSource->fetchAll('SELECT DISTINCT...', false)
#4 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2947): DboSource->read(Object(Checklist), Array)
#5 C:\xampp\htdocs\checklist\lib\Cake\Model\Model.php(2919): Model->_readDataSource('all', Array)
#6 C:\xampp\htdocs\checklist\app\Controller\ChecklistsController.php(66): Model->find('all', Array)
#7 [internal function]: ChecklistsController->index()
#8 C:\xampp\htdocs\checklist\lib\Cake\Controller\Controller.php(490): ReflectionMethod->invokeArgs(Object(ChecklistsController), Array)
#9 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(191): Controller->invokeAction(Object(CakeRequest))
#10 C:\xampp\htdocs\checklist\lib\Cake\Routing\Dispatcher.php(165): Dispatcher->_invoke(Object(ChecklistsController), Object(CakeRequest))
#11 C:\xampp\htdocs\checklist\app\webroot\index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2014-09-30 13:47:39 Error: Fatal Error (4): syntax error, unexpected 'echo' (T_ECHO), expecting ':' in [C:\xampp\htdocs\checklist\app\View\Blocks\admin_view.ctp, line 50]
2014-09-30 13:47:39 Error: [FatalErrorException] syntax error, unexpected 'echo' (T_ECHO), expecting ':'
Request URL: /checklist/admin/blocks/view/1
Stack Trace:
#0 C:\xampp\htdocs\checklist\lib\Cake\Error\ErrorHandler.php(203): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 50)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 50, Array)
#2 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 50, Array)
#3 C:\xampp\htdocs\checklist\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
